server/conf/application.conf

```diff
@@ -613,3 +613,8 @@ application_status_tracking_enabled = ${?CIVIFORM_APPLICATION_STATUS_TRACKING_EN
 # work as expected in production if there are multiple servers.
 feature_flag_overrides_enabled = false
 feature_flag_overrides_enabled = ${?FEATURE_FLAG_OVERRIDES_ENABLED}
+
+# If enabled, all assets (JS, CSS files) will be returned with long caching.
+# This setting should be used only by browser tests. Note that it is used
+# in dev mode where normally files served with caching disabled.
+enable_asset_caching_for_browser_tests = false
```

Comment:
I would probably prefer just name this "enable_dev_asset_caching", or something else that indicates that if this is true, it will enable it in all dev envs, not just in the test one.  Or, we could just have a flag that says "is_browser_test" and use that instead.